source/deploy/postgres-migration.rst

```diff
@@ -171,6 +171,8 @@ There is a specific unicode sequence that is `disallowed <https://www.postgresql
 
    DROP PROCEDURE IF EXISTS SanitizeUnsupportedUnicode;
 
+There is also a specific byte sequence value that is not allowed and will cause an ``invalid byte sequence for encoding 'UTF8': 0x00"`` error during the migration. To prevent this error, you can add the ``remove-null-characters`` clause to the text casting rules. However, since pgloader will modify the data on the fly, there may be differences between the tables (if any are affected) during the comparison phase.
```

Comment:
```suggestion
.. note::

     There is also a specific byte sequence value that is not allowed and will cause an ``invalid byte sequence for encoding 'UTF8': 0x00"`` error during the migration. To prevent this error, you can add the ``remove-null-characters`` clause to the text casting rules. However, since pgloader will modify the data on the fly, there may be differences between the tables (if any are affected) during the comparison phase.
```